### ISSUE TITLE
It seems Idea 2020.1 is not working properly with the plugin (or vice-versa, dunno)

### ISSUE THREAD

**Issue by igordonin-pareto** [google/google-java-format#486]
Tested only the block indentation and it's not being applied.

I did reinstall the plugin and restart the IDE with no effect.

**Comment by plumpy** [comment]
Did you enable the plugin in your project? It's not enabled by default.

**Comment by igordonin-pareto** [comment]
I did. Thanks for replying. If there's anything I can help with, let me know.

**Comment by sormuras** [comment]
Which Java version is used in your project?

**Comment by igordonin-pareto** [comment]
I'm currently using Open JDK 13.0.2.
Language level is 11.

**Comment by plumpy** [comment]
Which version of IntelliJ, which version of the plugin, and also some steps to reproduce would be helpful: some sample code you tried to format, what you did to do to trigger reformatting, does anything happen at all (i.e. does it get formatted incorrectly or you can't see anything happening at all) etc.

I just pushed a new version of the plugin, but it's not likely to fix your issues. It just updates the version of the `google-java-format` library; the plugin itself is unchanged.

**Comment by igordonin-pareto** [comment]
It was an issue with IntelliJ 2020.1.1, I believe. I just updated it to IntelliJ 2020.1.2 and all is working as expected.

I tried triggering reformat both by hitting `CTRL + ALT + L` and also using another plugin called Save Actions. It wasn't working either way.

The behavior was the opposite: the class was previously formatted, checked out from version control, and then when hitting the shortcut or saving changes, all the lines would be improperly formatted, clearly not applying google-format.

Anyways, all is fine with Idea's new version. Thanks for your time. I'll close the issue, but if you guys need additional info for any reason, I'll be glad to answer here.

Thanks
